src/components/Table/Table.tsx

```diff
@@ -0,0 +1,97 @@
+import React from "react";
+import { Box, Text } from "@radix-ui/themes";
+import {
+  FormatCellValue,
+  ColumnName,
+  RefactTableImpactLanguagesRow,
+  RefactTableData,
+} from "../../services/refact";
+import styles from "./Table.module.css";
+import { Spinner } from "../Spinner";
+
+export const Table: React.FC<{
+  refactTable: RefactTableData | null;
+}> = ({ refactTable }) => {
+  if (refactTable === null) {
+    return <Spinner />;
+  }
+  const refactImpactTable: RefactTableImpactLanguagesRow[] =
+    refactTable.table_refact_impact.data;
+  const convertedColumnNames: Record<ColumnName, string> = {
```

Comment:
could convertedColumnName and possibly formatCellValue could be defined outside of the scope of this function?
it's save redefining them every render :)

Reply:
fixed